hypervisor/arch/x86/guest/vmsr.c

```diff
@@ -332,6 +332,11 @@ static void init_msr_area(struct acrn_vcpu *vcpu)
 	vcpu->arch.msr_area.guest[MSR_AREA_TSC_AUX].value = vcpu->vcpu_id;
 	vcpu->arch.msr_area.host[MSR_AREA_TSC_AUX].msr_num = MSR_IA32_TSC_AUX;
 	vcpu->arch.msr_area.host[MSR_AREA_TSC_AUX].value = vcpu->pcpu_id;
+
+	vcpu->arch.msr_area.guest[MSR_AREA_IA32_MISC_ENABLE].msr_num = MSR_IA32_MISC_ENABLE;
```

Comment:
can we rename msr_num to msr_index? the later one is more accurate.

Reply:
ok